frontend/js/stats.js

```diff
@@ -179,6 +193,146 @@ const fillRunTab = async (selector, data, parent = '') => {
     }
 }
 
+const renderLogsInterface = (logsData) => {
+    const containerTemplate = `
+        <div class="title">
+            <i class="dropdown icon"></i><i class="server icon"></i> {{containerName}}
+            <div class="ui mini label">{{logCount}} log{{logPlural}}</div>
+        </div>
+        <div class="content">{{content}}</div>
+    `;
+
+    const logCardTemplate = `
+        <div class="ui card fluid">
+            <div class="content">
+                <div class="header">
+                    <div class="ui small labels">
+                        {{flowLabel}}
+                        {{typeLabel}}
+                        {{operationLabel}}
+                        {{phaseLabel}}
+                        {{idLabel}}
+                    </div>
+                </div>
+            </div>
+            {{commandContent}}
+            {{stdoutContent}}
+            {{stderrContent}}
+        </div>
+    `;
+
+    const stdoutTemplate = `
+        <div class="content">
+            <h5 class="ui header"><i class="file text outline icon"></i> Standard Output</h5>
+            <div class="ui segment stdout">
+                <div>{{stdout}}</div>
+            </div>
+        </div>
+    `;
+
+    const stderrTemplate = `
+        <div class="content">
+            <h5 class="ui header"><i class="exclamation triangle icon"></i> Standard Error</h5>
+            <div class="ui segment stderr">
+                <div>{{stderr}}</div>
+            </div>
+        </div>
+    `;
+
+    const logsElement = document.querySelector("#logs");
+    let accordionHTML = '<div class="ui styled accordion">';
+
+    Object.keys(logsData).forEach(containerName => {
+        const containerLogs = logsData[containerName];
+
+        let contentHTML = '';
+        containerLogs.forEach(logEntry => {
+            const typeIcon = logEntry.type === 'container_execution' ? 'cog' : 'play';
+            const typeTitle = escapeString(logEntry.type.replace('_', ' ').replace(/\b\w/g, l => l.toUpperCase()));
```

Comment:
is this supposed to only replace the first underscore? Can you please add a comment WHY you are writing this line to help understand the code in the future

Reply:
I added the following comment:
```js
// Make the type name more visually appealing by replacing underscores with spaces and capitalising the first letter of each word
```